app/models/agents/commander_agent.rb

```diff
@@ -28,7 +28,7 @@ class CommanderAgent < Agent
 
       - If you want to update a WeatherAgent based on a UserLocationAgent, you could use `'action': 'configure'` and set 'configure_options' to `{ 'location': '{{_location_.latlng}}' }`.
 
-      - In templating, you can use the variable `target` to refer to each target agent, which has the following attributes: #{AgentDrop.instance_methods(false).map { |m| "`#{m}`" }.to_sentence}.
+      - In templating, you can use the variable `target` to refer to each target agent, which has the following attributes: #{Agent::Drop.instance_methods(false).map { |m| "`#{m}`" }.to_sentence}.
```

Comment:
I am not sure if any Agent gems are using `AgentDrop` in a similar way, do you think it is possible/makes sense to also define the new class as `AgentDrop`? If not we should probably mention it in the changelog.

Reply:
GitHub code search didn't find any occurrence of it, but I agree it is worth a mention.